snowflake/README.md

```diff
@@ -27,12 +27,36 @@ datadog-agent integration install datadog-snowflake==2.0.1
 </div>
 
 ### Configuration
+1. Create a Datadog specific role and user to monitor Snowflake. In Snowflake, run the following to create a custom role with access to the ACCOUNT_USAGE schema.
 
-1. Edit the `snowflake.d/conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory to start collecting your snowflake performance data. See the [sample snowflake.d/conf.yaml][3] for all available configuration options.
-
-    **Note**: By default, this integration monitors the `SNOWFLAKE` database and `ACCOUNT_USAGE` schema.
+    <div class="alert alert-warning">Note: By default, this integration monitors the `SNOWFLAKE` database and `ACCOUNT_USAGE` schema.
     This database is available by default and only viewable by users in the `ACCOUNTADMIN` role or [any role granted by the ACCOUNTADMIN][8].
     
+    We recommend creating a `DATADOG` role with access to `ACCOUNT_USAGE`. Read more about controlling [ACCOUNTADMIN role][12] for more information.
+    </div>
```

Comment:
Markdown links will not work inside `div` tags on the documentation site. They must be code in HTML